mirrord/agent/src/steal/http/filter.rs

```diff
@@ -101,7 +101,7 @@ pub(super) async fn filter_task(
         Ok(mut reversible_stream) => {
             match HttpVersion::new(
                 reversible_stream.get_header(),
-                &H2_PREFACE[..MINIMAL_HEADER_SIZE],
+                H2_PREFACE.get(..MINIMAL_HEADER_SIZE).unwrap(),
```

Comment:
shouldn't that be okay to slice since it's const + const? if not, maybe we need to const something?

Reply:
maybe use https://docs.rs/const-str/latest/const_str/macro.to_char_array.html ?
(not very important though)